src/test/java/hw5/steps/AssertionSteps.java

```diff
@@ -0,0 +1,122 @@
+    public void shouldSeeMetalValueChangedInLogsBarOnTheDifferentElementPage(String label) {
+        SoftAssertions softAssert = new SoftAssertions();
+        Set<String> expectedLogMessages = new HashSet<>();
+        expectedLogMessages.add("metal: value changed to " + label);
+        Helper.textOfWebElementsContainsSetText(elementPage.getLogsLine(),
+                expectedLogMessages, softAssert);
+    }
+
+
+    @Then("I should see Colors: value changed to {string} "
+            + "in logs bar on the Different Element Page")
+    public void shouldSeeColorsValueChangedInLogsBarOnTheDifferentElementPage(String label) {
+        SoftAssertions softAssert = new SoftAssertions();
+        Set<String> expectedLogMessages = new HashSet<>();
+        expectedLogMessages.add("Colors: value changed to " + label);
+        Helper.textOfWebElementsContainsSetText(elementPage.getLogsLine(),
+                expectedLogMessages, softAssert);
+    }
+
+    @Then("User Table page should be opened")
+    public void userTablePageShouldBeOpened() {
+        assertThat(userTablePage.getTitle()).as("Title is not correct")
+                .isEqualTo("User Table");
+    }
+
+    @Then("Dropdown should be displayed in {string} row at Users Table")
+    public void dropdownShouldBeDisplayedInNumberAtUsersTable(String rowNumber) {
+        final boolean dropdownIsDisplayed = userTablePage.getDropdownInSpecifiedRow(rowNumber).isDisplayed();
+        assertThat(dropdownIsDisplayed).as("Can not find dropdown in the row number:" +rowNumber)
+                .isTrue();
+    }
+
+    @Then("Username should be displayed in {string} row as {string} at Users Table")
+    public void usernameShouldBeDisplayedInRowNumberAsUsernameAtUsersTable(
+            String rowNumber, String username) {
+        String usernameFromTable = userTablePage.getUsernameInSpecifiedRow(rowNumber).getText();
+        assertThat(usernameFromTable).as("Username doesn't match expected" + username)
+                .isEqualTo(username);
+    }
+
+    @Then("Checkboxes should be displayed in {string} row at Users Table on User Table Page")
+    public void checkboxesShouldBeDisplayedAtUsersTableOnUserTablePage(String row) {
+        boolean checkboxIsDisplayed = userTablePage.getCheckboxInSpecifiedRow(row).isDisplayed();
+        assertThat(checkboxIsDisplayed).as("Description is not correspond ")
+                .isTrue();
+    }
+
+    @Then("Log row has {string} text in log section")
+    public void logRowHasTextInLogSection(String expectedLog) {
+        SoftAssertions softAssertions = new SoftAssertions();
+        Helper.textOfWebElementsContainsSetText(userTablePage.getLogLines(),
+                new HashSet<>(Collections.singletonList(expectedLog)),
+                softAssertions);
+        softAssertions.assertAll();
+    }
+
+    @And("Description text under image in {string} row should be displayed as {string} with {string}")
```

Comment:
Please do not use `@And` and `@But` annotations in the step definitions class. This is not clear what is the purpose to the method

Reply:
Ok, thank you for your notes. I will avoid this annotations.